c/parallel/CMakeLists.txt

```diff
@@ -34,11 +34,7 @@ add_subdirectory(src/jit_templates)
 
 find_package(CUDAToolkit REQUIRED)
 
-# TODO Use static versions of cudart, nvrtc, and nvJitLink
 target_link_libraries(cccl.c.parallel PRIVATE
-  CUDA::cudart
-  CUDA::nvrtc
-  CUDA::nvJitLink
   CUDA::cuda_driver
```

Comment:
You would then want to use `CUDA::cudart_static` along with adding the other two back.

Reply:
I've done this with commits c522e47ea1fbdb89c04b1f9b90d4b85cbbbe4c9f and 8789fb761190854c75d5767cfba5392e421945b6.